.github/workflows/deploy-main.yml

```diff
@@ -13,19 +13,19 @@ jobs: # We need two separate builds as each is going to a different ECR repo
       id-token: write
     uses: elastic-ipfs/shared-workflows/.github/workflows/lambda-shared-build.yaml@main
     secrets:
-      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID_PROD }}
       aws_role_id: ${{ secrets.AWS_ROLE_ID }}
-      aws_region: ${{ secrets.AWS_REGION }}
+      aws_region: us-west-2
```

Comment:
should we use variable?

Reply:
https://github.com/elastic-ipfs/event-delivery-lambda/pull/1#discussion_r930239640